src/main/java/com/mastery/java/task/dao/EmployeeDaoImpl.java

```diff
@@ -0,0 +1,190 @@
+package com.mastery.java.task.dao;
+
+import com.mastery.java.task.dto.Employee;
+import com.mastery.java.task.dto.Gender;
+import org.springframework.stereotype.Component;
+
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class EmployeeDaoImpl implements EmployeeDao {
+
+    private final DataSource dataSource;
+
+    public EmployeeDaoImpl(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+
+    @Override
+    public void save(Employee employee) {
+        String query = "INSERT INTO employee (first_name, last_name, departament_id, job_title, gender, date_of_birth) values (?, ?, ?, ?, ?, ?)";
```

Comment:
Please use named constant String in the top of the class to store SQL query

Reply:
Done